core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -175,6 +175,15 @@
    * @param conf raft journal configuration
    */
   private RaftJournalSystem(RaftJournalConfiguration conf) {
+    // Override election/heartbeat timeouts for single master cluster.
+    if (conf.getClusterAddresses().size() == 1) {
+      long electionTimeout = ServerConfiguration
+          .getMs(PropertyKey.MASTER_EMBEDDED_JOURNAL_SINGLE_MASTER_ELECTION_TIMEOUT);
```

Comment:
Are there consequences if someone were to add another master to the cluster?
- Should we just not support this?
- Should we update the docs to specify this behavior?

Reply:
updated the docs.